From 881d1213f77d16d6d37a9324f12bef3a2c407dfe Mon Sep 17 00:00:00 2001
From: Christophe Giboudeaux <christophe@krop.fr>
Date: Sun, 20 Mar 2022 15:14:54 +0100
Subject: [PATCH] CMake: Don't assume KDE_INSTALL_* variables are relative

ECM only makes sure KDE_INSTALL_FULL_BINDIR and KDE_INSTALL_FULL_LIBEXECDIR
contain absolute paths.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4c5d6c7..a475d88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -99,8 +99,8 @@ include(ECMCoverageOption)
 include(ECMEnableSanitizers)
 
 file(RELATIVE_PATH LIBEXEC_REL_PATH
-   "${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_BINDIR}"
-   "${CMAKE_INSTALL_PREFIX}/${KDE_INSTALL_LIBEXECDIR}")
+   "${KDE_INSTALL_FULL_BINDIR}"
+   "${KDE_INSTALL_FULL_LIBEXECDIR}")
 
 configure_file(
   ${CMAKE_CURRENT_SOURCE_DIR}/hotspot-config.h.cmake
-- 
2.35.1

